build/AzurePipelinesTemplates/WindowsAppSDK-PackNuget-Steps.yml

```diff
@@ -0,0 +1,166 @@
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation arm64'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildBinaries_release_arm64"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation arm64'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildBinaries_release_anycpu"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation x64'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildAndTestMRT_release_x64"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation x86'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildAndTestMRT_release_x86"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+- task: DownloadPipelineArtifact@2
+  displayName: 'Download Foundation arm64'
+  inputs:
+    artifactName: "drop_TransportPackage_BuildAndTestMRT_release_arm64"
+    targetPath: '$(Build.SourcesDirectory)\BuildOutput'
+
+- task: PowerShell@2
+  name: StageFiles
+  inputs:
+    filePath: 'BuildAll.ps1'
+    arguments: -Platform "x86,x64,arm64" -Configuration "release" -AzureBuildStep "StageFiles"
+
+- task: CopyFiles@2
+  displayName: 'Copy symbols to artifact staging directory'
+  condition: always()
+  inputs:
+    sourceFolder: $(Build.SourcesDirectory)\BuildOutput\FullNuget
+    contents: |
+      **\*.pdb
+    targetFolder: $(Build.ArtifactStagingDirectory)\symbols
+
+- task: PublishSymbols@2
+  displayName: 'Publish private symbols to internal server (without source indexing)'
+  inputs:
+    searchPattern: '$(Build.ArtifactStagingDirectory)\symbols\**\*.pdb'
+    symbolServerType: 'TeamServices'
+  # This ADO task does not support indexing of github sources currently :-(
+    indexSources: false
+    detailedLog: true
+  # There is a bug which causes this task to fail if LIB includes an inaccessible path (even though it does not depend on it).
+  # To work around this issue, we just force LIB to be any dir that we know exists.
+  env:
+    LIB: $(Build.SourcesDirectory)
+
+- task: PublishSymbols@2
+  displayName: 'Publish symbols to public server(without source indexing)'
+  inputs:
+    searchPattern: '$(Build.ArtifactStagingDirectory)\symbols\**\*.pdb'
+    symbolServerType: 'TeamServices'
+  # This ADO task does not support indexing of github sources currently :-(
+    indexSources: false
+    detailedLog: true
+  # # There is a bug which causes this task to fail if LIB includes an inaccessible path (even though it does not depend on it).
+  # # To work around this issue, we just force LIB to be any dir that we know exists.
+  # env:
+  #   LIB: $(Build.SourcesDirectory)
+  #   ArtifactServices_Symbol_AccountName: microsoftpublicsymbols
+  #   ArtifactServices_Symbol_PAT: $(WinSDKLab_microsoftpublicsymbols_PAT)
```

Comment:
What am I missing? The above 2 tasks seem to be the same, other than 1) the display names, 2) somehow the 2nd task dose not need the LIB workaround?

Reply:
I am not exactly sure. 
This was in the original
https://github.com/microsoft/WindowsAppSDK/blame/develop/build/AzurePipelinesTemplates/WindowsAppSDK-BuildFoundation-job.yml